collaborative_practice.js

```diff
@@ -22,18 +22,32 @@ const listUsers = (users) => {
     console.log(output);
 };
 
-const searchUsers = (firstName, lastName, users) => {
-    const output = users
-        .map(user => (user.firstName === firstName && user.lastName === lastName) 
-        ? `ID: ${user.id}, First name: ${user.firstName}, Last name: ${user.lastName}`
-        : "User not found")
-    
-    console.log(output)
+const searchUsers = (name, users) => {
+    if (name.length >= 2) {
+        let exists = false;
+        users
+            .forEach(user => {
+                if (user.firstName.toLowerCase() === name.toLowerCase() || user.lastName.toLowerCase() === name.toLowerCase()) {
```

Comment:
We have to check if the name is contained in the first or last name
Eg: "an" matches "Dan" and "Daniels"

Reply:
i checked, it doesn't show "Dan" or "Daniel" if input is "an"